### PR TITLE
Fei Reserve Factor Update

### PR DESCRIPTION
**Pull request by 0xrajath**
## Motivation
This includes an AIP for the Fei Reserve Factor Update
More details can be found on the governance forum: https://governance.aave.com/t/bgd-aave-v2-ethereum-fei-security-report/10251

### CODE REVIEW

File: content/aips/3D16C7B29565-AIP-FEI-RESERVE-FACTOR-UPDATE.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+created: 2022-10-18
+---
+
+# Simple Summary
+
+This proposal updates FEI’s Reserve Factor on Aave v2 Ethereum from 100% to 99%. As noted by [BGD](https://governance.aave.com/t/bgd-aave-v2-ethereum-fei-security-report/10251), the current configuration of FEI’s Reserve Factor at 100% does not accrue variable borrow debt and does not redirect the Reserve Factor fees to the Collector smart contract.
+
+# Abstract
+
+As noted [here](https://governance.aave.com/t/bgd-aave-v2-ethereum-fei-security-report/10251), the Reserve Factor (RF) on Aave is a percentage representing the proportion of the supplier’s yield that gets redirected to the protocol as a “fee”. For example, WETH on Aave v2 Ethereum has a 10% RF, so if the yield for suppliers would be 5% annually, 10% of it (0.5%) would go to the protocol itself, to a Collector contract and in the shape of aTokens.
```
Comment (sakulstra) [inline content/aips/3D16C7B29565-AIP-FEI-RESERVE-FACTOR-UPDATE.md:16]: Are you 100% sure this sentence makes sense? If you get 5% yield you get 5% yield not 4.5% - the Rf is applied before. Would probably just remove the example.

**Review by sakulstra — APPROVED** [review]

File: content/aips/3D16C7B29565-AIP-FEI-RESERVE-FACTOR-UPDATE.md (hunk 1 above)
Comment (0xrajath) [reply content/aips/3D16C7B29565-AIP-FEI-RESERVE-FACTOR-UPDATE.md:16]: This is actually taken from the BGD write-up: https://governance.aave.com/t/bgd-aave-v2-ethereum-fei-security-report/10251#context-2

But if the example doesn't make sense, i'll just remove it for clarity's sake.

File: content/aips/3D16C7B29565-AIP-FEI-RESERVE-FACTOR-UPDATE.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+On September 05, [Aave governance proposal 96](https://app.aave.com/governance/proposal/96/) was executed, freezing the FEI asset on Aave v2 Ethereum.
+
+Initially, the payload seemed fine and performed two actions:
+1. Call a freeze() function on the protocol for FEI.
+2. In order for suppliers of FEI to not have incentives and withdraw, “redirect” all yield on the borrowers’ side to the protocol’s Collector. This was done by setting the Reserve Factor configuration of FEI to 100% (maximum).
+
+Later, it was realized that the variable borrow interest did not properly accrue for FEI borrowers. After some debugging, it was realized that the issue was caused by a side effect of configuring any RF to exactly 100%.
+
+The solution to fix this, is to set the FEI reserve factor to 99% through this AIP.
```
[the remark this answers is not in the document]
Comment (0xrajath) [reply content/aips/3D16C7B29565-AIP-FEI-RESERVE-FACTOR-UPDATE.md]: Done

File: content/aips/3D16C7B29565-AIP-FEI-RESERVE-FACTOR-UPDATE.md (hunk 1 above)
Comment (0xrajath) [reply content/aips/3D16C7B29565-AIP-FEI-RESERVE-FACTOR-UPDATE.md]: Done

File: content/aips/3D16C7B29565-AIP-FEI-RESERVE-FACTOR-UPDATE.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+The full motivation for this AIP can be found [here](https://governance.aave.com/t/bgd-aave-v2-ethereum-fei-security-report/10251)
+
+# Specification
+
+A full specification of the payload contract and tests enabling this mandate can be found on the Llama Github [here](https://github.com/llama-community/aave-fei-rf-update)
+
+The Proposal Payload does the following:
+1. Sets FEI Reserve Factor from 100% to 99%.
+2. If liquidity is available to claim, then calls `swapAllAvailable()` on the [aFEI to aDAI Swapper Contract](https://etherscan.io/address/0x9A953AC1090C7014D00FD205D89c6BA1C219Af8b)
```
Comment (sakulstra) [inline content/aips/3D16C7B29565-AIP-FEI-RESERVE-FACTOR-UPDATE.md:42]: is claim the correct word here? probably better `withdraw`
```suggestion
2. If liquidity is available to withdraw, then calls `swapAllAvailable()` on the [aFEI to aDAI Swapper Contract](https://etherscan.io/address/0x9A953AC1090C7014D00FD205D89c6BA1C219Af8b)
```

**Comment by sakulstra** [comment]
@miguelmtzinf / @eboadom can someone do final review and merge?

File: content/aips/3D16C7B29565-AIP-FEI-RESERVE-FACTOR-UPDATE.md (hunk 3 above)
Comment (0xrajath) [reply content/aips/3D16C7B29565-AIP-FEI-RESERVE-FACTOR-UPDATE.md]: Done

**Pull request merged**